libraries/provision/ansible/playbooks/start-sgload.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: load_generators
+  any_errors_fatal: true
+
+  # Kill gateload
+  tasks:
+  - name: kill previous sgload screen
+    shell: screen -X -S sgload kill
+    ignore_errors: yes
+  - name: kill previous sgload_expvars screen
+    shell: screen -X -S sgload_expvars kill
+    ignore_errors: yes
+
+
+- hosts: load_generators
+  any_errors_fatal: true
+  become: yes
+  vars:
+    sync_gateway_node: "{{ hostvars[groups.sync_gateways[0]].ansible_host }}"
+
+  # Deploy sgload script and start it
+  tasks:
+    - debug: msg={{ sync_gateway_node }}
+    - name: copy the run sgload script
+      copy: src=files/run_sgload.sh dest=/home/centos/ owner=centos group=centos mode=0755
+    - name: start sgload in screen
```

Comment:
Fixed in https://github.com/couchbaselabs/mobile-testkit/pull/743/commits/05065bbb825d33b36bbbb016e379e7a1e568f7aa -- was dead code